app/js/controllers/foyer/logement.js

```diff
@@ -9,6 +12,14 @@ angular.module('ddsApp').controller('FoyerLogementCtrl', function($scope, $http,
     $scope.logementTypes = logementTypes;
     $scope.locationTypes = locationTypes;
 
+    $scope.cityStartsWith = function cityStartsWith(prefix) {
+        return logement.adresse.nomCommune.indexOf(prefix.toUpperCase()) === 0;
```

Comment:
Because jshint asks it

Le mar. 23 juin 2015 10:31, Matti Schneider <notifications@github.com> a
écrit :

> In app/js/controllers/foyer/logement.js
> <https://github.com/sgmap/mes-aides-ui/pull/115#discussion_r33019671>:
>
> > @@ -9,6 +12,14 @@ angular.module('ddsApp').controller('FoyerLogementCtrl', function($scope, $http,
> >      $scope.logementTypes = logementTypes;
> >      $scope.locationTypes = locationTypes;
> >
> > +    $scope.cityStartsWith = function cityStartsWith(prefix) {
> > +        return logement.adresse.nomCommune.indexOf(prefix.toUpperCase()) === 0;
>
> Why === rather than ==? indexOf will always return a Number.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sgmap/mes-aides-ui/pull/115/files#r33019671>.
>